src/components/pages/ImageClassification/ImageClassificationTable/ImageClassificationObservationTable.js

```
@@ -0,0 +1,429 @@
+                    <StyledTd>{rowIndex++}</StyledTd>
+                    {subIndex === 0 && (
+                      <>
+                        <TdWithHoverText
+                          rowSpan={numSubRows}
+                          data-tooltip={file.original_image_name}
+                          onClick={() => handleImageClick(file)}
+                          cursor={file.classification_status.status === 3 ? 'pointer' : 'default'}
+                        >
+                          <Thumbnail imageUrl={file.thumbnail} />
+                        </TdWithHoverText>
+                      </>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)